fe/src/main/java/org/apache/doris/clone/DynamicPartitionScheduler.java

```diff
@@ -210,7 +215,8 @@ public void createOrUpdateRuntimeInfo(String tableName, String key, String value
             reservePartitionKeyRange = Range.closedOpen(lowerBound, upperBound);
         } catch (AnalysisException e) {
             // keys.size is always equal to column.size, cannot reach this exception
-            LOG.warn("Keys size is not equal to column size. Error={}", e.getMessage());
+            LOG.warn("Keys size is not equal to column size. Error={}, db: {}, table: {}", e.getMessage(),
+                    db.getFullName(), olapTable.getName());
```

Comment:
maybe we should remove the warning, because dynamic_partition.start is default to Integer.MIN_VALUE, so we may got a AnalysisException like this `Error=errCode = 2, detailMessage = date literal [5877471-07-26 00:00:00] is invalid`

Reply:
OK, I will check this offset to make it reasonable.

Reply:
I just make a check that if `start offset == Integer.MIN_VALUE`, it will not drop any partition.
And for adding partition, I just leave it to user to make sure the value is reasonable.